test/plugin_helper/test_event_emitter.rb

```diff
@@ -48,4 +49,32 @@ class Dummy < Fluent::Plugin::TestBase
 
     d1.terminate
   end
+
+  test 'should not have event_emitter_router' do
+    d0 = Dummy0.new
+    assert !d0.respond_to?(:event_emitter_router)
+  end
+
+  test 'should have event_emitter_router' do
+    d0 = Dummy.new
```

Comment:
This line should be confusable on first glance.
We should use to store `d` for Dummy class' instance.

Reply:
Ah, it should be so.